spec/requests/admin/configs_spec.rb

```diff
@@ -237,13 +237,29 @@
           expect(SiteConfig.main_social_image).to eq(expected_image_url)
         end
 
+        it "only updates the main_social_image if given a valid image URL" do
+          expected_image_url = "![logo_lowres]https://dummyimage.com/300x300"
```

Comment:
**nitpick (non-blocking):** Total nitpick, but this isn't really expected, since the spec uses a `not_to change` assertion. I'd maybe just call it `invalid_image_url`.

Reply:
This is not a nitpick at all, this is great! I've made this change.